darc/darc.go

```diff
@@ -964,6 +973,10 @@ func (id Identity) GetPublicBytes() []byte {
 		return buf
 	case 4:
 		return id.EvmContract.Address[:]
+	case 5:
+		buf := elliptic.Marshal(id.ECDSA.PublicKey.Curve, id.ECDSA.PublicKey.X, id.ECDSA.PublicKey.Y)
+		//TODO: add error check here?
```

Comment:
What kind of error check can you do here? The `GetPublicBytes` doesn't return an error, so all you could do is `panic`.